client/src/lib/mockedMeals.js

```diff
@@ -0,0 +1,45 @@
+import meal1 from "../images/meal1.png";
+import meal2 from "../images/meal2.png";
+import meal3 from "../images/meal3.png";
+import chef1 from "../images/profilePic.png";
+
+const meals = [
+    {
+        id: 123,
+        title: "4 specialty rolls",
+        mealPic: meal1,
```

Comment:
shouldn't this be a url in from of a string pointing to a s3 resource

Reply:
Yes it will be but this was just a quick set up I did using the assets we were given. It looks like setting that up will be part of my next issue of setting up the chef profile.